app/src/userpages/components/StreamPage/Show/StatusView/index.jsx

```diff
@@ -91,17 +101,17 @@ function StatusView(props: Props) {
                     onChange={onChange}
                     onBlur={onCommit}
                     disabled={disabled}
+                    preserveLabelSpace
```

Comment:
What does `preserveLabelSpace` do exactly?

Reply:
It basically includes the label to be part of the input element i.e. label is not just positioned absolutely on top of the input without taking space.